src/keyboards/user_keyboards.py

```
@@ -5,57 +5,95 @@
-        [InlineKeyboardButton(text=T["to_main"], callback_data="back_to_main")]
-    ])
+    """Клавиатура 'Назад' к главному меню"""
+    return InlineKeyboardMarkup(inline_keyboard=[[InlineKeyboardButton(text=T["to_main"], callback_data="back_to_main")]])
+
 
 def kb_account_list(accounts, selected_email=None):
     """Клавиатура со списком аккаунтов"""
     buttons = []
-    
+
     for email, username, is_temp, temp_password in accounts:
```

Ruff (0.12.2)

Loop control variable `username` not used within loop body

Rename unused `username` to `_username`

(B007)